app/controllers/authenticate_controller.rb

```diff
@@ -83,7 +83,8 @@ def authenticator_input
       username:           params[:id],
       credentials:        request.body.read,
       client_ip:          request.ip,
-      request:            request
+      request:            request,
+      encoded_token:      ! params[:encoded].nil?
```

Comment:
how will the request look like? can you share the URL?

Reply:
```https://conjur-domain/authn/cucumber/admin/authenticate?encoded=<whatever>```
If encoded is mention the response will be encoded.

Reply:
Also that's why I prefer to keep the `params[:encoded].nil?` even though the codeclimate says that it's redundant. In my opinion, it clarifies that encoded attribute depends on whether encoded parameter has been submitted

Reply:
well, in that case it seems that it is better to use just `params[:encoded]` because in your implementation a user might set `?encoded=false` and you will encode it.

if yo use `encoded_token: params[:encoded]` then in case it is there you will get the given value and if it is not present then it will be nil (so false)

Reply:
> that's why I prefer to keep the params[:encoded].nil? even though the codeclimate says that it's redundant

but it is redundant. it is nil in case it is nil so it is false

Reply:
@liavyona i answered your question here